app/controllers/users/sessions_controller.rb

```diff
@@ -16,6 +16,23 @@ class Users::SessionsController < Devise::SessionsController
   #   super
   # end
 
+  # Singing in with authentication token (needed when signing in automatically
+  # from another website)
+  def auth_token_create
+    user = User.find_by_email(params[:user_email])
+    if user.authentication_token == params[:user_token][0..-2]
+      sign_in(:user, user)
```

Comment:
I also thought that would be a good idea, but then I read it's safe enough... But I guess It's better to be more safe than sorry :).